tests/px_terminal_test.py

```diff
@@ -10,8 +10,8 @@
 def test_to_screen_lines_unbounded():
     procs = [testutils.create_process(commandline="/usr/bin/fluff 1234")]
     assert px_terminal.to_screen_lines(procs, None) == [
-        "  PID COMMAND USERNAME   CPU RAM COMMANDLINE",
-        "47536 fluff   root     0.03s  0% /usr/bin/fluff 1234"
+        "  PID COMMAND USERNAME CPU CPU TIME RAM COMMANDLINE",
+        "47536 fluff   root      0% 0.03s  0% /usr/bin/fluff 1234"
```

Comment:
The spacing in this second line is mis-aligned with the heading. `/usr/...` should start right below `COMMANDLINE`.